WordPress/src/main/res/values/strings.xml

```diff
@@ -4373,13 +4373,20 @@ translators: %s: Select control option value e.g: "Auto, 25%". -->
 
     <!-- Jetpack full plugin install onboarding -->
     <string name="jetpack_individual_plugin_support_onboarding_title">Please install the full Jetpack plugin</string>
-    <!-- translators: %1$s = the current site address (e.g. "https://wordpress.com/"), %2$s = the name of the plugin (e.g. "Jetpack Backup"), %3$s = the text "full Jetpack plugin" -->
-    <string name="jetpack_full_plugin_install_onboarding_description">%1$s is using the %2$s plugin, which doesn’t support all features of the app yet.\n\nPlease install the %3$s to use the app with this site.</string>
+    <!-- translators:
+        %1$s = the current site address (e.g. "https://wordpress.com/"),
+        %2$s = can be either "the <plugin name> plugin" (e.g. "the Jetpack Backup plugin") OR "individual plugins"
+        %3$s = the text "full Jetpack plugin" -->
+    <string name="jetpack_full_plugin_install_onboarding_description">%1$s is using %2$s, which don’t support all features of the app yet.\n\nPlease install the %3$s to use the app with this site.</string>
```

Comment:
np: not sure if it was discussed already, thinking of `which don't support...` should we consider singular vs plural? wdyt?

Reply:
Thanks for catching that! As discussed in DM, it was decided to use `which don't support`.